src/parser.js

```diff
@@ -1183,8 +1183,7 @@ export default class Parser {
       const enumerations = new ArrayElement();
 
       _.forEach(parameter.enum, (value, index) => {
-        const e = new Type();
-        e.content = value;
+        const e = new Type(value);
```

Comment:
This fix is pretty trivial. That's because the constructor for an ArrayElement will correctly refract the given values inside the array. Whereas the setter for `e.content` does not handle this.